plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryClient.java

```diff
@@ -188,9 +216,14 @@ public TableInfo getCachedTable(Duration viewExpiration, TableInfo remoteTableId
         return materializationCache.getCachedTable(this, query, viewExpiration, remoteTableId);
     }
 
+    public String getParentProjectId()
```

Comment:
It worth adding comments to `getParentProjectId` & `getProjectId` methods for explaining the differences.

Reply:
Hi @ebyhr thanks for taking a look. Yeah sounds good, I'll some comments there.